src/internal/lib/connectors/services/water/ReportingService.js

```diff
@@ -0,0 +1,24 @@
+const ServiceClient = require('shared/lib/connectors/services/ServiceClient');
+const got = require('got');
+
+class ReportingService extends ServiceClient {
+  /**
+   * Requests a report from the API
+   * @param request {Object} The request object
+   * @returns {Request} Request body
+   */
+  getReport (request) {
```

Comment:
I think I would expect only the needed params to be passed in here, e.g. maybe:
```
getReport (userId, reportIdentifier)
```
This would make the connector code more flexible in case it's needed elsewhere

Reply:
Sure thing. This is done in 5deb30d1